ansible/setup.yml

```diff
@@ -84,6 +84,13 @@
       tags:
         - dev-tools
 
+    - name: Increase ulimit
+      pam_limits:
+        domain: epoch
+        limit_type: soft
+        limit_item: nofile
+        value: 24576
```

Comment:
I see this is taken from https://github.com/aeternity/epoch/blob/v0.9.0/hooks/pre_start.sh#L3
Ok.

Reply:
yes